datadog-service/0.15.0/README.md

```diff
@@ -0,0 +1,198 @@
+```
+Example:
+```bash
+$ keptn add-resource --project="podtatohead" --stage="hardening" --service="helloservice" --resource=./quickstart/sli.yaml --resourceUri=datadog/sli.yaml
+$ keptn add-resource --project="podtatohead" --stage="hardening" --service="helloservice" --resource=./quickstart/slo.yaml --resourceUri=slo.yaml
+```
+
+4. Configure Keptn to use datadog SLI provider
+Use keptn CLI version [0.15.0](https://github.com/keptn/keptn/releases/tag/0.15.0) onwards
+```bash
+$ keptn configure monitoring datadog --project <project-name>  --service <service-name>
+```
+
+5. Trigger delivery
+```bash
+$ keptn trigger delivery --project=<project-name> --service=<service-name> --image=<image> --tag=<tag>
+```
+Example:
+```bash
+$ keptn trigger delivery --project=podtatohead --service=helloservice --image=docker.io/jetzlstorfer/helloserver --tag=0.1.1
+```
+Observe the results in the Keptn bridge
+## Compatibility Matrix
+
+*Please fill in your versions accordingly*
+
+| Keptn Version    | [datadog-service Docker Image](https://github.com/keptn-sandbox/datadog-service/pkgs/container/datadog-service) |
+|:----------------:|:----------------------------------------:|
+|       0.11.4      | ghcr.io/keptn-sandbox/datadog-service:0.1.0 |
+|       0.11.4      | ghcr.io/keptn-sandbox/datadog-service:0.2.0 |
+|       0.15.0      | ghcr.io/keptn-sandbox/datadog-service:0.15.0 |  
+
+datadog-service version will match Keptn version starting from 0.15.0 version of Keptn.
+
+## Installation
+
+```bash
+$ export DD_API_KEY="<your-datadog-api-key>" DD_APP_KEY="<your-datadog-app-key>" DD_SITE="datadoghq.com" 
+# cd datadog-service
+$ helm install datadog-service ./helm --set datadogservice.ddApikey=${DD_API_KEY} --set datadogservice.ddAppKey=${DD_APP_KEY} --set datadogservice.ddSite=${DD_SITE}
+```
+Tell Keptn to use datadog as SLI provider for your project/service
+```bash
+$ keptn configure monitoring datadog --project <project-name>  --service <service-name>
+```
+
+This should install the `datadog-service` together with a Keptn `distributor` into the `keptn` namespace, which you can verify using
+
+```console
+kubectl -n keptn get deployment datadog-service -o wide
+kubectl -n keptn get pods -l run=datadog-service
+```
+### Up- or Downgrading
+
+Adapt and use the following command in case you want to up- or downgrade your installed version (specified by the `$VERSION` placeholder):
+
+```bash
+helm upgrade datadog-service ./helm --set datadogservice.ddApikey=${DD_API_KEY} --set datadogservice.ddAppKey=${DD_APP_KEY} --set datadogservice.ddSite=${DD_SITE}
+```
+
+### Uninstall
+
+To delete a deployed *datadog-service* helm chart:
+
+```bash
+helm uninstall datadog-service
+```
+
+## Development
+
+Development can be conducted using any GoLang compatible IDE/editor (e.g., Jetbrains GoLand, VSCode with Go plugins).
+
+It is recommended to make use of branches as follows:
+
+* `master` contains the latest potentially unstable version
```

Comment:
I thought we were advocating that new repos use main rather than master.  It's nasty to try to change existing repos so we may have to live with the "master" terminology there but good if we do not propagate the problem into new repos.

Reply:
I believe [the service template](https://github.com/keptn-sandbox/keptn-service-template-go) I used to create the boilerplate for datadog-service set `master` as the main branch. Newer services e.g., [sumologic-service](https://github.com/keptn-sandbox/sumologic-service) has the default branch set to `main`

Reply:
confirm, it's coming from the service-template. I think it does not add any value to the artifacthub entry, so I would recommend removing development specific sections here.

Reply:
Removed. Thank you for the suggestion.